contents/founders/cracked-manifesto.md

```diff
@@ -0,0 +1,79 @@
+
+Hm, sounds pretty ripe for a toxic monoculture of tech bros doing whatever they want while the founders arbitrarily pick new pet projects based on what they last saw DHH tweet, right?
+
+No! The following are definitely not prerequisites to being cracked:
+
+- Fancy companies on your resume. You don’t need to have worked at a FAANG/MAMAA/shit hot SV-based company. We welcome cracked engineers who held together failing companies - we like their paranoia. 
+- Being an asshole. People need to want to work with you - saying ‘screw you, I’m doing this anyway’ is uncracked behavior if you can’t back it up. 
+- There is no number of years of experience that correlates with crackedness, in either direction. 
+- You built a side project used by 1m people. Side projects are often great signs of crackedness/ownership, but they don’t have to be commercially successful - they just have to be interesting. 
+
+What’s cool about adopting this mentality is that it has noticeably increased the diversity of people applying to work with us at PostHog, as it ignores some of the traditional markers of status when it comes to job applications.
+
+### How to manage a cracked team
+
+A lot of traditional management behavior doesn’t apply here, and can actively slow down or demotivate your team.
+
+**Be cracked yourself**
+
+You need to demonstrate the behavior you want to see. Cracked people are extremely attuned to ‘do as I do’ not ‘do as I say’. This means spending a significant amount of time doing IC work. Do the stuff that arrogant senior people would find ‘boring’. Write tests. Respond to customer support queries. Go weirdly deep into some piece of analysis. Do something badly that’s not your job so you at least understand it better. 
+
+**Spend most of your time generating enthusiasm**
+
+It is unbelievably easy to find reasons not to do stuff at any kind of organization, including startups. Inertia is the default state of most businesses. Overcoming this inertia is mainly reliant on the enthusiasm of people to make new/cool things happen far more than their raw intelligence or competence. 
+
+This is why you should spend more time motivating people than trying to teach them new skills or making them do training - if they are cracked and staying motivated, they will train themselves. I don’t think I’ve yet read a blog post by someone I admire and learned from that says ‘I got good because I attended blah blah special leadership course on being great.’
+
+**Get out of the way**
+
+Cracked people are deeply intrinsically motivated. This means that extrinsic motivation, while still present in some form (we’ve all got to pay bills yeah), is way less important. Help cracked engineers take ownership of their own projects by encouraging them to make the decisions about what to do next. If they ask for help - ie. they actively want you to get in the way - ask questions about what they think should happen next, rather than giving them the answers. 
```

Comment:
```suggestion
Cracked people are deeply intrinsically motivated. This means that extrinsic motivation, while still present in some form (we’ve all got to pay bills, right?), is way less important. 

Help cracked engineers take ownership of their own projects by encouraging them to make the decisions about what to do next. If they ask for help – i.e. they actively want you to get in the way – ask questions about what they think should happen next, rather than giving them the answers. 
```